build_unix/build.sh

```diff
@@ -4,3 +4,8 @@ cd build
 cmake ..
 make
 cd ..
+mkdir -p build-ui/
+~/Qt5.6.*/5.6*/gcc*/bin/qmake -o build-ui/
```

Comment:
initially, because wanted to use qt 5.6, so that it would include this [patch](https://codereview.qt-project.org/#/c/139775/13/src/widgets/kernel/qopenglwidget.cpp), but i don't think thats needed anymore. Ill confirm that once i get back from college tonight.